extensions/amp-nexxtv-player/validator-amp-nexxtv-player.protoascii

```diff
@@ -42,22 +42,22 @@ tags: {  # <amp-nexxtv-player>
     value: "api"
     value: "static"
   }
-  attrs: {
-    name: "data-origin"
-    value_url: {
-      protocol: "https"
-      protocol: "http"
-      allow_empty: true
-    }
-  }
   attrs: {
     name: "data-streamtype"
-    value: "album"
+    value: "audioalbum"
     value: "audio"
     value: "live"
     value: "playlist"
-    value: "playlist-marked"
     value: "video"
+    value: "radio"
+    value: "set"
+    value: "collection"
```

Comment:
Please bring back `value: "album"` and `value: "playlist-marked"`. Otherwise any existing document that uses those values will be rendered invalid.

Reply:
done